src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -457,6 +457,8 @@
   private Builder() {
     @if xsettingsClass.hasDefaultInstance
       super(defaultChannelProviderBuilder().build());
+    @else
+      super((InstantiatingChannelProvider) null);
```

Comment:
Yes, because otherwise the call is ambiguous (there is another single-arg constructor that takes ServiceApiSettings).